scripts/install-all-subpackages.ts

```diff
@@ -0,0 +1,26 @@
+import { execSync, type SpawnSyncReturns } from "node:child_process";
+import { findDirWithFileRecursive } from "./find.js";
+
+export async function installAllSubpackages(
+  filename: string,
+  command: string
+): Promise<void> {
+  for (const project of await findDirWithFileRecursive(filename)) {
+    if (project == ".") continue;
+    try {
+      console.log(`+ ${command} [${project}]`);
+      execSync(command, {
+        cwd: project,
+        stdio: "inherit",
+      });
```

Comment:
This shouldn't block. Can we use `exex()` or `spawn()` instead?

Reply:
That can cause the output of the commands to overwrite each other, is that ok?
Or should I save the outputs to a variable and print them when the command is done?

Reply:
Hm, I realize I accidentally commented in `install-all-subpackes.ts`, but wanted to comment in `check-tsc.ts`, which takes longer in this branch, because previously it was running two checks in parallel (`xargs -P 2`). The `npx tsc --noEmit` commands shouldn't print anything unless there are errors, so I don't think we have to worry about them colliding.

Reply:
(As for `install-all-subpackages.ts`, I think it's okay to stick with `execSync()` for now, **it's still faster**!)